Sources/CoreAardvark/Logging/ARKExceptionLogging.m

```diff
@@ -33,7 +33,7 @@
 ARKUncaughtExceptionHandlerGetter ARKGetUncaughtExceptionHandler = NSGetUncaughtExceptionHandler;
 ARKUncaughtExceptionHandlerSetter ARKSetUncaughtExceptionHandler = NSSetUncaughtExceptionHandler;
 
-NSLock * ARKGetUncaughtExceptionLogDistributorsLock()
+NSLock * ARKGetUncaughtExceptionLogDistributorsLock(void)
```

Comment:
I think you'll need to put this newer style declaration behind an `#if` statement checking that we're using a later Xcode version. Right now Aardvark still supports Xcode 11, so we can't land changes that break that version.

Reply:
Thanks, I included another commit that applies this last fix for Xcode 12 an newer